package.json

```diff
@@ -51,6 +51,7 @@
     "immutable": "^4.0.0-rc.14",
     "lodash-es": "^4.17.21",
     "material-design-icons-iconfont": "^6.1.0",
+    "ngx-long-press2": "^2.0.0",
```

Comment:
For long press feature I use https://github.com/AnthonyNahas/ngx-long-press2 
It is actively maintained and seems secure to use